setup.py

```diff
@@ -48,6 +48,7 @@
     extras_require={
         'all': [
             'autopep8',
+            'flake8==3.7.8',
```

Comment:
Even though we use internal APIs, let’s not pin this. Otherwise we’ll end up conflicting with people’s own Flake8 versions.

We should just track latest the best we can I reckon

Reply:
done